src/event_loop.rs

```diff
@@ -490,6 +491,18 @@ impl ActiveEventLoop {
             platform: self.p.owned_display_handle(),
         }
     }
+
+    /// Returns the current cursor position relative to the top-left hand corner of the desktop.
+    ///
+    /// See [`Window::outer_position`] for more information about the coordinates.
+    ///
+    /// ## Platform-specific
+    ///
+    /// - **iOS / Android / Wayland / Orbital / Web**: Unsupported.
+    #[inline]
+    pub fn cursor_position(&self) -> Result<PhysicalPosition<f64>, ExternalError> {
```

Comment:
It confused me that the return type of this is not `PhysicalPosition<i32>` like the return type of `Window::outer_position`, but I guess it matches the `CursorMoved` event - apparently cursors can be positioned on a sub-pixel scale, didn't know that!